crates/vizia_core/src/views/textbox.rs

```diff
@@ -861,6 +838,21 @@ where
 
             _ => {}
         });
+
+        if let Some(msg) = event.take::<TextEventInternal>() {
+            let TextEventInternal::Reset = msg;
```

Comment:
Should this be an `if let`?

Reply:
It would be an irrefutable if-let, so this lints better. This essentially just serves to use the variable. When the enum has more than one variant, this can be replaced with if-let.

Reply:
Ah I see, I did wonder if it was some kind of placeholder for now.